data/json/recipes/recipe_deconstruction.json

```
@@ -812,11 +812,32 @@
       [ [ "turret_chassis", 1 ] ]
     ]
   },
+  {
+    "result": "broken_turret_riot",
+    "type": "uncraft",
+    "skill_used": "electronics",
+    "difficulty": 4,
+    "time": "1 h",
+    "using": [ [ "soldering_standard", 10 ] ],
+    "qualities": [ { "id": "SCREW", "level": 1 } ],
+    "components": [
+      [ [ "ai_module", 1 ] ],
+      [ [ "gun_module", 1 ] ],
+      [ [ "targeting_module", 1 ] ],
+      [ [ "identification_module", 1 ] ],
+      [ [ "sensor_module", 1 ] ],
+      [ [ "m1014", 1 ] ],
```

```suggestion
      [ [ "m203", 16 ] ],
```

This should match the capacity of the pseudo m203 array, yes?

---

Yes, I thought I fixed that a while back. Thanks